.github/workflows/pull-requests.yml

```diff
@@ -61,6 +61,9 @@ jobs:
     runs-on: ubuntu-latest
     env:
       UTOPIA_SHA: ${{ github.sha }}
+      AUTH0_CLIENT_ID: KB7euFO46rVYeOaWmrEdktdhAFxEO266
+      AUTH0_ENDPOINT: enter.utopia.app
```

Comment:
This does expose our staging environment to the public, I thought this was going to go in the settings?

Reply:
oh crap, I misunderstood our conversation, once we settled that these are not private keys, I thought they don't need to be in the Github Secrets either (because the client will have these strings compiled into them anyways)